fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/converters/PropertyConverter.java

```diff
@@ -128,7 +128,10 @@ public static String getPropertyNameFromPredicate(final NamespaceRegistry namesp
         }
 
         final String rdfLocalname = predicate.getLocalName();
-
+        if (rdfLocalname.isEmpty()) {
+          LOGGER.info("predicate local name is null");
+          throw new RepositoryException("Illegal predicate local name ");
```

Comment:
Extra space in the message here. Also, this should be a `RepositoryRuntimeException` or a more specific subtype of that.